elements/pl-file-editor/pl-file-editor.js

```diff
@@ -162,10 +168,32 @@ window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
           })
         );
       }
+
+      var keyBindingList = ['Default', 'Vim', 'Emacs', 'Sublime', 'VSCode'];
+      var keyBindSelect = that.modal.find('#modal-' + uuid + '-keyboardHandler');
+      keyBindSelect.empty();
+      for (const index in keyBindingList) {
+        var keyBind = 'ace/keyboard/' + keyBindingList[index].toLowerCase();
+
+        keyBindSelect.append(
+          $('<option>', {
+            value: keyBind,
+            text: keyBindingList[index],
+            selected: localStorage.getItem('pl-file-editor-keyboardHandler') === keyBind,
+          })
+        );
+      }
     });
     that.modal.modal('show');
     sessionStorage.setItem('pl-file-editor-theme-current', that.editor.getTheme());
     sessionStorage.setItem('pl-file-editor-fontsize-current', that.editor.getFontSize());
+    if (localStorage.getItem('pl-file-editor-keyboardHandler')) {
+      sessionStorage.setItem(
+        'pl-file-editor-keyboardHandler-current',
+        localStorage.getItem('pl-file-editor-keyboardHandler')
+      );
+    }
```

Comment:
I believe the conditional is not required - `getItem` will just return `null` if it's not set (or if it's set to null).
```suggestion
    sessionStorage.setItem(
      'pl-file-editor-keyboardHandler-current',
      localStorage.getItem('pl-file-editor-keyboardHandler')
    );
```

Reply:
Somehow, An error with appearing after removing the if statement. 

![截屏2022-12-13 下午7 49 23](https://user-images.githubusercontent.com/92755550/207485184-fb5fcb84-55c9-4670-b9da-2381e4de8cf8.png)
 this if statement

Reply:
Ah! I keep forgetting the fact that `null` gets turned into `"null"` when you put it into web storage 😅 this looks fine!